app/views/application/_flashes.html.erb

```diff
@@ -1,8 +1,10 @@
 <% if flash.any? %>
   <div class="center-div">
-    <div class="flash" role="alert">
+    <div role="alert">
       <% user_facing_flashes.each do |key, value| -%>
-        <div class="font-bold flash-<%= key %>"><%= value %></div>
+        <div class="font-bold <%= key == 'notice' ? 'flash-notice' :
+          'flash-alert' %>"><%= value %>
```

Comment:
**suggestion:** Remove the duplication

How would you feel about the following change? 🙂 This way, you would remove the word "flash" duplication. 🔄 One of the rules of simple design is to ensure there is no duplication. ✨ Making this change would be a lovely refactoring. 💡

```erb
        <div class="font-bold flash-<%= key %>"><%= value %>
```

Reply:
When I attempt this, the styling on the alert disappears.
![Styling is gone](https://github.com/Esseme/rumblr/assets/3297508/7ee73f6e-b043-4dd8-989b-8ceeac3f5770)

Reply:
I tried this refactoring on my machine, and everything looked okay.

I raised #167 with the changes.

Look and see how those changes differ from what you have done.

Feel free to merge that pull request or raise another one with your changes.